test/xml-generator/ParameterFrameworkConfiguration.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ParameterFrameworkConfiguration SystemClassName="Test" ServerPort="5066" TuningAllowed="false">
+    <SubsystemPlugins>
```

Comment:
I do not know, I think we discouraged upper case file names in the coding style.